.circleci/config.yml

```diff
@@ -243,4 +269,17 @@ workflows:
             - unitTests
             - integrationTests
             - referenceTests
-            - docker
\ No newline at end of file
+            - docker
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
does the timezone/timing of this matter?

Reply:
No, I don't actually care when it runs as long as it's about every day.